_script/lib/verify_couches.rb

```diff
@@ -0,0 +1,40 @@
+class Couches 
+    def initialize(location = "../_couches")
+        @couch_dir = location
```

Comment:
You can use [glob](http://ruby-doc.org/core-2.0.0/Dir.html#method-c-glob) to get a list of just markdown files.